### PR TITLE
global: excludes option added

### PR DESCRIPTION
**Pull request by hachreak**
* Adds a new option "excludes" to add a list of regular expression of
  files to exclude from the checks.

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: kwalitee/cli/check.py [hunk 1]
```diff
@@ -185,8 +186,11 @@ def _ensure_directory(filename):
 
     def _format_errors(args):
         filename, errors = args
-        return error_template.format(filename=filename, errors='\n'.join(
-            errors if len(errors) else no_errors))
+        if errors is False:
```
Comment (jirikuncar) [inline kwalitee/cli/check.py]: `if not errors:`?

**Comment by jirikuncar** [comment]
Please check the failing tests. Thanks

File: kwalitee/cli/check.py (hunk 1 above)
Comment (hachreak) [inline kwalitee/cli/check.py]: The `check_file()` normally returns 2 kinds of result:
 - `errors = [ ]` interpreted as `no error`
 - `errors = [...]` (not empty array) interpreted as `there is at least a error`

But, I needed a third state to say: `the file is excluded`.
So, the easy way was return a `errors = False` value.

The `if not errors` is `True` also for empty array and I can't distinguish the case `no error` from the case `the file is excluded`.

Comment (jirikuncar) [inline kwalitee/cli/check.py]: ok

**Comment by jirikuncar** [comment]
comes from https://github.com/inveniosoftware/invenio/issues/3022

**Comment by jirikuncar** [comment]
@hachreak you can rebase now. 

--
PS: Please have a look on failing docs build. Thanks

**Comment by hachreak** [comment]
do you talk about `sphinx-build -vnNW docs _build`?
Because locally it works fine.

**Pull request merged**